### DISCUSSION TITLE
Will restarting the pulsar broker node cause the cluster to become unhealthy?

### DISCUSSION THREAD

**Discussion by bingbb47** [milvus-io/milvus#23809] [Q&A and General discussion] [answered]
We have identified an issue with expiration policies in pulsar, need to restart the brokers. 
so will the milvus cluster be affected？

**Comment by yhmo** [comment, marked as answer]
Milvus service is not available during you restarting the pulsar broker, and back to service after the pulsar broker successfully restarted.

**Reply by bingbb47** [reply]
> Milvus service is not available during you restarting the pulsar broker, and back to service after the pulsar broker successfully restarted.

how about rolling restart brokers?  can also make milvus not available ？

**Reply by yhmo** [reply]
Not available
